### ISSUE TITLE
Navbar reloads everytime

### ISSUE THREAD

**Issue by Pancham1603** [BVNCodeTech/medsched#12]
The dashboard navbar reloads everytime when you switch pages, ig only the page content should refresh

**Comment by sortedcord** [comment]
Yeah this was something that I was able to fix with Django, I made navbar into a static block and used that block in all the other pages. https://github.com/sortedcord/MathMaster/blob/master/templates/base.html#L84 Here I made a base html. So it leads to faster load times. I'm not sure if you can do this with Flask tho...

**Comment by Pancham1603** [comment]
yes yes, that's possible in jinja, we create a base and in other files we just mention {% extends base.html %} in the other files
